### ISSUE TITLE
Showing CKEditor shortcuts in the toolbar

### ISSUE THREAD

**Issue by mgoldbeck** [neos/neos-ui#1601]
### Description

The most shortcuts of CKEditor are working but the most editors are not aware of them.

Especially the ones for
* text formattting https://docs.ckeditor.com/ckeditor4/latest/guide/dev_shortcuts.html#text-styling
* and Link editing https://docs.ckeditor.com/ckeditor4/latest/guide/dev_shortcuts.html#rich-text
should be useful

Complete List of the shortcuts https://docs.ckeditor.com/ckeditor4/latest/guide/dev_shortcuts.html

I would suggest to show a button for the shortcuts in the Editor Toolbar

**Comment by mstruebing** [comment]
I would suggest to make something like a shortcut-overview when pressing `?` without being in any input element

**Comment by dimaip** [comment]
Isn't this now implemented? Reopen if not. /cc @mstruebing @mgoldbeck
